### PR TITLE
Update Task 0 and Task 1 for Fall 2022

### PR DESCRIPTION
**Pull request by anirudh-chakravarthy**
The following changes were made:

- Added motivation and introduction for assignment. The objective was unclear in the last offering. Let me know if it is still not clear enough.
- Updated task 0 instructions. Removed the older Q0.2 (since there was some confusion about it last semester) and added it as a hint for implementation.
- Updated Task 1. Made the visualization intervals more clear since it was lacking last semester. Added question numbers alongside TODOs in code as well. Set pre-trained flag to be True by default (it was previously False).

Needs discussion:

- [x] Can we remove the functionality part in Q1.1? Since students have already implemented it, they are already aware of the functionality it is performing.
- [x] Should we add a conda environment file? Last time, I used the latest libraries and it worked fine. 
- [x] Is the location of paper readings fine in the README?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+- TAs: [Anirudh Chakravarthy](https://anirudh-chakravarthy.github.io/), [Sai Shruthi Balaji](https://www.linkedin.com/in/sai-shruthi-balaji/), [Vanshaj Chowdhary](https://www.linkedin.com/in/vanshajchowdhary/), and [Nikos Gkanatsios](https://nickgkan.github.io/).
 
 - We will be keeping an updated FAQ on piazza. Please check the FAQ post before posting a question.
-- Due date: March 29th, 2021 at 11:59pm EST.
+- Due date: Oct 3rd, 2022 at 11:59pm EST.
 - Total points: 100
 
-In this assignment, we will learn to train object detectors in the *weakly supervised* setting, which means you're going to train object detectors without bounding box annotations!
+## Introduction
+In this assignment, we will learn to train object detectors without bounding box annotations! First, in task 1, we will use classification models and examine their backbone features for object localization cues. In task 2, we will train object detectors in the *weakly supervised* setting, which means we're going to train an object detector without bounding box annotations!
```
Comment (shruthibalaji2307) [inline README.md]: "In this assignment, we will learn to train object detectors with only image level annotations, and no bounding box annotations!"

File: task_1.py [hunk 2]
```diff
@@ -196,7 +191,7 @@ def main():
 
     
     
-    # TODO: Create loggers for wandb - ideally, use flags since wandb makes it harder to debug code.
+    # TODO (Q1.3): Create loggers for wandb - ideally, use flags since wandb makes it harder to debug code.
 
 
     for epoch in range(args.start_epoch, args.epochs):
```
Comment (shruthibalaji2307) [inline task_1.py:193]: The next line, "adjust_learning_rate" is deprecated according to [this Piazza thread](https://piazza.com/class/kyiph2whvsp5i8/post/159_f12). We can remove it.

File: README.md (hunk 1 above)
Comment (anirudh-chakravarthy) [reply README.md]: Done

File: task_1.py (hunk 2 above)
Comment (anirudh-chakravarthy) [reply task_1.py:193]: Done

**Pull request merged**